### PR TITLE
Payment and vote tests

### PR DESCRIPTION
**Pull request by nckackerman**
100% line coverage for the 2 files these unit tests cover

### CODE REVIEW

File: api/src/main/java/com/nicknathanjustin/streamercontracts/contracts/ContractServiceImpl.java [hunk 1]
```diff
@@ -19,10 +19,10 @@
     @Override
     public ContractModel createContract(@NonNull final UserModel proposer, @NonNull final UserModel streamer, @Nullable final String game, @NonNull final String description) {
         final Timestamp creationTimestamp = new Timestamp(System.currentTimeMillis());
-        // Add 10 days to the current timestamp to create the expires timestamp
+        // Add 3 days to the current timestamp to create the expires timestamp
         final Calendar calendar = Calendar.getInstance();
         calendar.setTime(creationTimestamp);
-        calendar.add(Calendar.DAY_OF_WEEK, 10);
+        calendar.add(Calendar.DAY_OF_WEEK, 3);
```
Comment (m1sterc001guy) [inline api/src/main/java/com/nicknathanjustin/streamercontracts/contracts/ContractServiceImpl.java]: Make this a constant? I generally think its bad practice to have numbers hard coded in the code.

File: api/src/main/java/com/nicknathanjustin/streamercontracts/votes/VoteServiceImpl.java [hunk 2]
```diff
@@ -144,8 +142,8 @@ private void validateContract(@NonNull final ContractModel contractModel) {
             throw new IllegalArgumentException("Voting logic not yet implemented for community contracts. ContractId " + contractModel.getId());
         }
 
-        if (!contractModel.isAccepted()) {
-            throw new IllegalArgumentException("Cannot vote on an unaccepted contract. ContractId " + contractModel.getId());
+        if (!contractModel.isAccepted() && !isContractExpired(contractModel)) {
+            throw new IllegalArgumentException("Cannot vote on an unaccepted contract that hasnt expired yet. ContractId " + contractModel.getId());
```
Comment (m1sterc001guy) [inline api/src/main/java/com/nicknathanjustin/streamercontracts/votes/VoteServiceImpl.java]: After the contract has expired, you can vote on an unaccepted contract?

File: api/src/test/java/com/nicknathanjustin/streamercontracts/contracts/ExpiredContractsSqsHandlerTest.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        expiredContractsSqsHandler.settleExpiredDonations(new Object());
+
+        verify(mockContractService, never()).settlePayments(any(), anyBoolean());
+    }
+
+    @Test
+    public void settleExpiredDonations_expiredDonations_settlesExpiredContracts() {
+        final Set<ContractModel> expiredContracts = new HashSet<>();
+        final int numberOfExpiredContracts = 15;
+        for (int i=0; i<numberOfExpiredContracts; i++) {
```
Comment (m1sterc001guy) [inline api/src/test/java/com/nicknathanjustin/streamercontracts/contracts/ExpiredContractsSqsHandlerTest.java]: Nit: add spaces between "=" and "<"

File: api/src/test/java/com/nicknathanjustin/streamercontracts/contracts/ExpiredContractsSqsHandlerTest.java [hunk 4]
```diff
@@ -0,0 +1,56 @@
+package com.nicknathanjustin.streamercontracts.contracts;
```
Comment (m1sterc001guy) [inline api/src/test/java/com/nicknathanjustin/streamercontracts/contracts/ExpiredContractsSqsHandlerTest.java]: Out of curiosity, how are you getting your code coverage number?

File: api/src/main/java/com/nicknathanjustin/streamercontracts/contracts/ContractServiceImpl.java (hunk 1 above)
Comment (nckackerman) [reply api/src/main/java/com/nicknathanjustin/streamercontracts/contracts/ContractServiceImpl.java]: Sure.... I wonder who added the magic number in the first place ;)

File: api/src/main/java/com/nicknathanjustin/streamercontracts/votes/VoteServiceImpl.java (hunk 2 above)
Comment (nckackerman) [reply api/src/main/java/com/nicknathanjustin/streamercontracts/votes/VoteServiceImpl.java]: Good catch. The exception message was off. Should change 'vote' to 'settle'. Settle being payout funds. Fixed.

File: api/src/test/java/com/nicknathanjustin/streamercontracts/contracts/ExpiredContractsSqsHandlerTest.java (hunk 4 above)
Comment (nckackerman) [reply api/src/test/java/com/nicknathanjustin/streamercontracts/contracts/ExpiredContractsSqsHandlerTest.java]: Intellij lets us run unit tests 1 of 3 ways:
- run 
- debug
- run with coverage. 

The last option pops up a useful UI to show coverage statistics.

File: api/src/test/java/com/nicknathanjustin/streamercontracts/contracts/ExpiredContractsSqsHandlerTest.java (hunk 3 above)
Comment (nckackerman) [reply api/src/test/java/com/nicknathanjustin/streamercontracts/contracts/ExpiredContractsSqsHandlerTest.java]: done

File: api/src/main/java/com/nicknathanjustin/streamercontracts/contracts/ContractServiceImpl.java (hunk 1 above)
Comment (m1sterc001guy) [reply api/src/main/java/com/nicknathanjustin/streamercontracts/contracts/ContractServiceImpl.java]: Probably me :)

File: api/src/test/java/com/nicknathanjustin/streamercontracts/contracts/ExpiredContractsSqsHandlerTest.java (hunk 4 above)
Comment (m1sterc001guy) [reply api/src/test/java/com/nicknathanjustin/streamercontracts/contracts/ExpiredContractsSqsHandlerTest.java]: Gotcha, I'll have to give this a try at some point.

**Pull request merged**